AsyncDisplayKit/ASCollectionView.h

```diff
@@ -79,6 +79,30 @@
 @property (nonatomic, assign) CGFloat leadingScreensForBatching;
 
 /**
+ *  Perform a batch of updates asynchronously, optionally disabling all animations in the batch. You can call it from background 
+ *  thread (it is recommendated) and the UI collection view will be updated asynchronously. The asyncDataSource must be updated 
```

Comment:
Review my final comments on the old PR — there is a typo here and I'm also worried about the explicit advice to call on a background thread.  Very interested to hear your thoughts on this, but I was pretty confident we should not be even /supporting/ edits on a background thread — so probably not recommend them :)